js/renovation/viz/core/tooltip.tsx

```diff
@@ -187,49 +210,35 @@ export class TooltipProps {
 
   @OneWay() customizeTooltip?: CustomizeTooltipFn;
 
-  @OneWay() canvas = {
-    left: 0, top: 0, right: 0, bottom: 0, width: 0, height: 0,
-  };
+  @OneWay() canvas? = DEFAULT_CANVAS;
```

Comment:
Why you don't use a Canvas type here and for DEFAULT_CANVAS variable?

Reply:
Fixed